translations/nm-tray_de.ts

```diff
@@ -0,0 +1,168 @@
+        <translation>Allgemeines</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="949"/>
+        <source>Interface</source>
+        <comment>Active connection information</comment>
+        <translation>Schnittstelle</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="951"/>
+        <source>Hardware Address</source>
+        <comment>Active connection information</comment>
+        <translation>Hardware-Adresse</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="953"/>
+        <source>Driver</source>
+        <comment>Active connection information</comment>
+        <translation>Treiber</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="955"/>
+        <source>Speed</source>
+        <comment>Active connection information</comment>
+        <translation>Übertragungsrate</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="957"/>
+        <source> Kb/s</source>
+        <translation> Kb/s</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="959"/>
+        <source>unknown</source>
+        <comment>Speed</comment>
+        <translation>unbekannt</translation>
+    </message>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="962"/>
+        <source>Security</source>
+        <comment>Active connection information</comment>
+        <translation>Sicherheit</translation>
+    </message>
+</context>
+<context>
+    <name>Tray</name>
+    <message>
+        <location filename="../src/tray.cpp" line="165"/>
+        <source>Enable Networking</source>
+        <translation>Netzwerk aktivieren</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="166"/>
+        <source>Enable Wi-fi</source>
+        <translation>Drahtlose Verbindungen aktivieren</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="168"/>
+        <source>Connection information</source>
+        <translation>Verbindungsinformationen</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="170"/>
+        <source>About</source>
+        <translation>Über</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="172"/>
+        <source>Quit</source>
+        <translation>Beenden</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="226"/>
+        <source>%1 about</source>
+        <translation>%1 über</translation>
+    </message>
+    <message>
+        <location filename="../src/tray.cpp" line="227"/>
+        <source>This is the about nm-tray!</source>
+        <translatorcomment>Pimped this a little bit. Hope that&apos;s okay.</translatorcomment>
```

Comment:
Yes, it's completely OK. As you probably noticed, I didn't care much about the wording .... For now I just want to know if the whole work wasn't useless and the tool is/will be/ usable.